frontend/src/packages/articles/articles-api.package.ts

```diff
@@ -101,6 +101,19 @@ class ArticleApi extends HttpApi {
 
     return await response.json<ArticleWithAuthorType>();
   }
+
+  public async delete(id: number): Promise<ArticleWithAuthorType> {
+    const response = await this.load(
+      this.getFullEndpoint(ArticlesApiPath.$ID, { id: String(id) }),
+      {
+        method: 'DELETE',
+        contentType: ContentType.FORM_DATA,
```

Comment:
Why `contentType` is `FORM_DATA`? Shouldn't it be `ContentType.JSON` 🤔

Reply:
Actually, missed the payload field, and there was an error with ContentType.JSON. But now resolved, thanks)